app/models/user.rb

```diff
@@ -1,13 +1,7 @@
 class User < ApplicationRecord
   validates :email, uniqueness: true, presence: true
+  validates :password, confirmation: true, presence: true
   validates_presence_of :address, :city, :state, :zip, :password, :name
```

Comment:
I decided not to validate the presence of password_confirmation here. It seemed redundant since we're validating the confirmation so in order for that to pass the password_confirmation must be present. Open to thoughts on that!

Reply:
So, instead of `validates :password_confirmation` you just included a `:true` confirmation along with the password validation? That makes sense to me!

Reply:
That's right!

Reply:
Nice! I was experimenting with combining different validations to see what it would give me. I'm glad the solution was simple